test/markdownlint-cli2-test-cases.js

```diff
@@ -444,6 +444,14 @@ const testCases =
     "usesRequire": true
   });
 
+  testCase({
+    "name": "markdownlint-cli2-config-option-with-extends",
+    "args": [ "--config", "configs/custom.markdownlint-cli2.jsonc", "**/*.md" ],
+    "exitCode": 0,
+    "cwd": "markdownlint-cli2-config-option-with-extends",
+    "env": onlyRunViaExec
```

Comment:
If this is set, only one of the snapshot files should get updates, I think?

Reply:
Removed other snapshots.